### PR TITLE
use 64bit for seed hash + fix incorrect terminology

### PR DESCRIPTION
**Pull request by 0xAdk**
this isn't tested, but nothing crazy was done so this should work

### CODE REVIEW

**Comment by connorhsm** [comment]
This PR will be marked as completed and reviewed in the days leading up to the next update where we can merge this in.
This change would update the location for all seeds, so ideally this will be lined up with a map wipe, and we do not want to accidentally merge it before then.

**Review by connorhsm — DISMISSED** [review]
Ready to merge, right before next map wipe.

**Comment by connorhsm** [comment]
@0xAdk Can you please review the merge of master into this branch?

File: server/server.cpp [hunk 1]
```diff
@@ -13250,11 +13249,11 @@ int main() {
 
                                     // Get the substr from one after the seed delim
                                     std::string seed { emailAndSeed.substr( seedDelimPos + 1 ) };
-                                    char *sSeed = SettingsManager::getStringSetting("seedSalt", "default salt");
-                                    std::string seedSalt { sSeed };
+                                    char *sSeed = SettingsManager::getStringSetting("seedPepper", "default pepper");
```
Comment (0xAdk) [inline server/server.cpp:13252]: is there a reason sSeed is created if it's just used to initialize a string and deleted afterwards?

**Review by connorhsm — APPROVED** [review]

**Comment by 0xAdk** [comment]
lgtm :+1:

**Pull request merged**